### PR TITLE
Codefreeze for 0.21.4

### PR DESCRIPTION
**Pull request by IaroslavTitov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+## 0.21.4
+
+### Miscellaneous
+
+- Fixed integ tests [#328](https://github.com/pulumi/pulumi-pulumiservice/issues/328)
```
Comment (komalali) [inline CHANGELOG.md:8]: I think the result was that this was not just fixing the tests? There were meaningful bug fixes included in this change. Can we please capture what we fixed for end users?

Comment (IaroslavTitov) [reply CHANGELOG.md:8]: Updated to include bug https://github.com/pulumi/pulumi-pulumiservice/issues/319, which was also fixed during the PR that fixed integ tests.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,13 @@
...
+- Fixed Environment Get function by fixing the resource's Read method [#319](https://github.com/pulumi/pulumi-pulumiservice/issues/319)
```
Comment (komalali) [inline CHANGELOG.md:9]: Should this be under `Bug Fixes`?

**Review by komalali — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (IaroslavTitov) [reply CHANGELOG.md:9]: Ah, you're right, I just added below without much thought

**Pull request merged**